# Candida Albicans RNA-seq project
This project is part of a genomics and bioinformatics class at Georgetown University with the objective of analyzing RNA-seq data from _Candida Albicans_.

# Project Description
The question we are looking to answer is why C. Albicans, a normally commensal yeast, can become pathogenic in certain environmental conditions and causes various human pathologies, such as urinary tract, genital, mucosal and blood infections.

One hypothesis for why this transition may be occurring is that they only become pathogenic in nutrient-deficient environments. To test this hypothesis, we have grown C. Albicans in Thiamine-present and Thiamine-absent media and used RNAseq to analyze their transcriptomic profiles in these conditions. 

# Data Collection and Raw Data
Original FastQ file sequencing data was generated by Dr. Ronda Rolfes and generously allowed us to use it. 
The data collected was paired-end sequencing data, and three biological replicates were made per treatment group. 

From the experimental workflow, 12 raw data files were generated:
WTA1_1.fq.gz, WTA1_2.fq.gz
WTB1_1.fq.gz, WTB1_2.fq.gz
WTC1_1.fq.gz, WTC1_2.fq.gz

WTA2_1.fq.gz, WTA2_2.fq.gz
WTB2_1.fq.gz, WTB2_2.fq.gz
WTC2_1.fq.gz, WTC2_2.fq.gz

The naming conventions are as follows: A/B/C correspond to biological replicates, A1/B1/C1 are from the Thiamine-present group, A2/B2/C2 are from the Thiamine-absent group, and _1,_2 are matched read pairs. 

# Trimming and Data Cleaning
The FastQ data was processed by running it through FastQC to guide the subsequent trimming strategy. Afterwards, Trimmimotatic was used to remove any adapter content, remove the first 15 bases, eliminate trailing bases with a low quality score, remove potential reads with low average quality socres and remove any reads that were less than 75 basepairs. This processed data was ran through FastQC to analyze results of the trimming.

# Reference Genome Mapping
To map RNAseq reads, we utilized a reference genome downloaded from NCBI (GCF_000182965.3). This reference genome was indexed using Bowtie2 to generate an indexed map, and then aligned against that map.
